### PR TITLE
cloud: ovirt: add developer README.rst

### PR DESCRIPTION
**Pull request by machacekondra**
##### SUMMARY

Add developer README file.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

ovirt

##### ANSIBLE VERSION

```
2.3.0
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina

File: lib/ansible/modules/cloud/ovirt/README.md
Comment (mykaul) [inline lib/ansible/modules/cloud/ovirt/README.md]: oVirt/RHV perhaps?

File: lib/ansible/modules/cloud/ovirt/README.md
Comment (mykaul) [inline lib/ansible/modules/cloud/ovirt/README.md]: start with -> an 'ovirt_' prefix

File: lib/ansible/modules/cloud/ovirt/README.md
Comment (mykaul) [inline lib/ansible/modules/cloud/ovirt/README.md]: Rephrase: 'Version 3 of the API is not supported.'

File: lib/ansible/modules/cloud/ovirt/README.md
Comment (mykaul) [inline lib/ansible/modules/cloud/ovirt/README.md]: things -> properties, functions, features

File: lib/ansible/modules/cloud/ovirt/README.md
Comment (mykaul) [inline lib/ansible/modules/cloud/ovirt/README.md]: remove 'like'

**Review by mwperina — APPROVED** [review]
👍

**Comment by mattclay** [comment]
CI is now passing. There was a bug in the module validator which was causing CI to fail.

**Comment by mattclay** [comment]
@machacekondra New documentation should use reStructuredText instead of Markdown. We're slowly migrating away from using Markdown in most places.

**Comment by mattclay** [comment]
The validate-module errors are wrong. I'll fix the validator and re-run CI.

**Comment by mattclay** [comment]
The one remaining CI failure appears to be valid:

```
lib/ansible/modules/cloud/ovirt/README.rst:180:0: (python) expected an indented block
```

**Comment by machacekondra** [comment]
Thanks for help @mattclay

**Comment by machacekondra** [comment]
shipit

**Review by mwperina — APPROVED** [review]
👍

**Comment by machacekondra** [comment]
@ryansb

**Comment by ryansb** [comment]
Waiting on docs approval from @dharmabumstead

**Comment by dharmabumstead** [comment]
Review in progress. ETA ~Thursday 6/15 by 6pm PDT.

File: lib/ansible/modules/cloud/ovirt/README.rst
Comment (dharmabumstead) [inline lib/ansible/modules/cloud/ovirt/README.rst]: Please reword this; I'm unclear what you're trying to say here.

**Review by dharmabumstead — CHANGES_REQUESTED** [review]

**Comment by machacekondra** [comment]
Thank you for review, I've changed the requested paragraph.

**Comment by machacekondra** [comment]
@dharmabumstead please re-review.

**Comment by machacekondra** [comment]
@dharmabumstead Can you please re-review?

**Review by mwperina — APPROVED** [review]
Looks good to me, thanks! 👍

**Comment by mwperina** [comment]
@dharmabumstead Can you please re-review?

**Review by dharmabumstead — APPROVED** [review]
Docs are OK with a few minor edits. Still needs core team sign off before merging. Thanks @machacekondra!

**Pull request merged**
